### PR TITLE
ViewWin32 Cursor Prop Type Expanded

### PR DESCRIPTION
**Pull request by tkamireh**
Expands the type for the ViewWin32 Cursor Prop and adds a test component

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5154)

### CODE REVIEW

File: docs/api/react-native-windows.datepicker._constructor_.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ constructor(props: IDatePickerProps);
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  props | <code>IDatePickerProps</code> |  |
+|  props | [IDatePickerProps](./react-native-windows.idatepickerprops.md) |  |
```
Comment (lenahong) [inline docs/api/react-native-windows.datepicker._constructor_.md:19]: What's with all the md file changes?

Comment (tkamireh) [reply docs/api/react-native-windows.datepicker._constructor_.md:19]: This is a consequence of people not running `yarn api` more often. These should just be formatting changes, but they've been lingering around for a while and no one has gotten around to checking them in.

Comment (tkamireh) [reply docs/api/react-native-windows.datepicker._constructor_.md:19]: @NickGerleman does that right?

Comment (NickGerleman) [reply docs/api/react-native-windows.datepicker._constructor_.md:19]: I think the format of api documenter changed, but there weren't any api changes, so api extractor didn't force people to regenerate docs

**Pull request merged**